packages/secrets/src/keyring/README.md

```diff
@@ -0,0 +1,169 @@
+# keyring
+
+keyring is a native Node.js module for accessing and managing OS credential storage.
+
+## Compatibility
```

Comment:
Should we update the compatibility matrix to be current?
We have added an x86 build for Linux.
Also, we don't yet have an s390x build.